CONTRIBUTING.md

```diff
@@ -96,15 +106,22 @@ or run manually:
 docker run --runtime=nvidia --rm -it -v ${PWD}:/addons -w /addons gcr.io/tensorflow-testing/nosla-cuda10.1-cudnn7-ubuntu16.04-manylinux2010 /bin/bash
 export TF_NEED_CUDA=1
 ./configure.sh  # Links project with TensorFlow dependency
+```
+
+Run individual tests:
 
+```bash
 bazel test -c opt -k \
 --test_timeout 300,450,1200,3600 \
 --crosstool_top=//build_deps/toolchains/gcc7_manylinux2010-nvcc-cuda10.1:toolchain \
 --test_output=all \
 --jobs=1 \
-//tensorflow_addons/...
```

Comment:
Let's not delete the command for running all the bazel tests. We can have both, i.e., `//tensorflow_addons/...` and the newer `tensorflow_addons/<package>:<py_test_name>`

Reply:
Yes - good point. I fixed that.